<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainClass</class>
 <widget class="QWidget" name="CMainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RAPIT Platform</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Project File</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="butLoad">
        <property name="text">
         <string>Load Project File...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="butSave">
        <property name="text">
         <string>Save Project File...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="butPreference">
        <property name="text">
         <string>Preference</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1454</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="butFluMod">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>UltraFluMod</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butSPARATS">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>UltraSPARATS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butNSMod">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>UltraNSMod</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butPICA">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>UltraPICA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butUNIC">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>UNIC-A Unified CFD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butProfiling">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>UltraProfiling</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butChemRateMod">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>UltraChemRateMod</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CMain.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>butLoad</sender>
   <signal>clicked()</signal>
   <receiver>CMainClass</receiver>
   <slot>slotLoadProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>butSave</sender>
   <signal>clicked()</signal>
   <receiver>CMainClass</receiver>
   <slot>slotSaveProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>butPreference</sender>
   <signal>clicked()</signal>
   <receiver>CMainClass</receiver>
   <slot>slotPreference()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1872</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotLoadProject()</slot>
  <slot>slotSaveProject()</slot>
  <slot>slotPreference()</slot>
 </slots>
</ui>
